### ISSUE TITLE
Same privilege id used for 'Append' and 'Append To' on query results

### ISSUE THREAD

**Issue by liorabel** [MscrmTools/MsCrmTools.AccessChecker#4]
I have an entity set up where applied security roles grant 'Append To' but not 'Append'. Despite this, the query results in Access Checker mark them both as green.
After a bit of digging, it appears they both have the same Privilege Id. This looks like something is misconfigured somewhere as the tool is duplicating one and neglecting the other.

![image](https://user-images.githubusercontent.com/3285032/42998379-e27f98b4-8c11-11e8-93e7-ce7f189080e5.png)

**Comment by MscrmTools** [comment]
Can you check with this fixed assembly?
[MsCrmTools.AccessChecker.zip](https://github.com/MscrmTools/MsCrmTools.AccessChecker/files/2213852/MsCrmTools.AccessChecker.zip)

**Comment by liorabel** [comment]
Two things.
First, I just discovered an error in my original test. Seems there was a sharing privilege applied that I did not find before. Now the red/green colours are correct with both versions of the DLL.

BUT!
It seems there was actually a bug present which is only partially described in my original issue. The Privilege Ids were indeed duplicated. Here's a comparison of the results between the old and new DLL versions.
Old DLL with identical ids:
![image](https://user-images.githubusercontent.com/3285032/43008059-f64f4c98-8c31-11e8-9451-8ef6b8064e30.png)

New DLL with different ids:
![image](https://user-images.githubusercontent.com/3285032/43008080-06c4df02-8c32-11e8-83b6-3809e024b24c.png)

So it seems I should both apologise for reporting a bug that did not exist but at the same time also congratulate you for fixing a bug I reported.
So thank you and my apologies. :)
